ee_tests/src/booster_ee_int_tests/booster_trigger_cd.spec.ts

```diff
@@ -25,13 +45,114 @@ describe('Triggers the CD Build (Jenkins):', () => {
   });
 
   afterEach(async () => {
-    support.writeScreenshot('target/screenshots/booster_trigger_cd_success.png');
+    support.writeScreenshot('target/screenshots/booster_modify_src_complete.png');
```

Comment:
Please, keep the name of the screenshots  aligned with the name of the scenario.